setup.py

```diff
@@ -10,7 +10,7 @@
     author="Department for Levelling Up, Housing and Communities",
     description="Provides access to functionality common to creating a data dashboard.",
     name="gov_uk_dashboards",
-    version="6.8.0",
+    version="7.1.0",
```

Comment:
Why has the version number increased from 6.8.0 to 7.1.0?

Reply:
Is it because you have merged other PR's in which had a higher version number than 6.8.0?

Reply:
Hi Kate - my version wasn't up to date, so there was a bit of merge conflict sorting out, leading to this package number